### PR TITLE
Upgrade FHIR module to FHIR2 which is the new path for OpenMRS

### PR DESCRIPTION
**Pull request by slubwama**
Upgrade FHIR module to FHIR2 which is the new path for OpenMRS
https://app.asana.com/0/1133167201254915/1196158922422521

### CODE REVIEW

**Comment by slubwama** [comment]
@solemabrothers

File: pom.xml [hunk 1]
```diff
@@ -82,7 +82,7 @@
 		<dataintegrityVersion>4.4.3</dataintegrityVersion>
 		<emrapiVersion>1.29.0</emrapiVersion>
 		<eventVersion>2.7.0</eventVersion>
-		<fhirVersion>1.20.0</fhirVersion>
+		<fhir2Version>1.0.0</fhir2Version>
```
Comment (solemabrothers) [inline pom.xml:85]: @slubwama this looks like a downgrade of the module, why are we doing this.

Comment (solemabrothers) [reply pom.xml:85]: @slubwama I have seen the difference.

Comment (slubwama) [reply pom.xml:85]: @solemabrothers its not. Its a new module which has  lower versioning numbers

**Pull request merged**